front_end/css/styles.css

```diff
@@ -559,3 +558,15 @@ module {
 .test-content pre {
   font-size: 1em;
```

Comment:
These changes address markdown styling inside instructions. (Some overlapping styles made it so inline code and code blocks were being styled differently inside an exercise's instructions, this brings them into harmony)